### DISCUSSION TITLE
kimai2 Saas offers

### DISCUSSION THREAD

**Discussion by Michael-Schaer** [kimai/kimai#3457] [On-Premise: Installation & Updates] [answered]
Is there any known organization that offers kimai2 as a complete service and takes care of all the updates and maintenance?

I'm not too lazy to host it myself, i'm asking for a friend 😛

**Comment by kevinpapst** [comment, marked as answer]
Is it seriously so complicated to find it 😢 I guess I have to re-style the website ... 

GitHub about:
<img width="694" alt="Bildschirmfoto 2022-08-02 um 13 43 57" src="https://user-images.githubusercontent.com/533162/182366593-ab5e609d-d902-4e4f-82bb-ac5655577d4f.png">

www.kimai.org
<img width="1236" alt="Bildschirmfoto 2022-08-02 um 13 42 24" src="https://user-images.githubusercontent.com/533162/182366455-34c82a90-3524-4a75-9c95-8181e1450a38.png">

Did you miss those or have you been too lazy to search?
Did you google search for "kimai saas" not show up anything?
These are serious questions, because my income depends on people being able to find it.

The answer is yes: I do offer that service at www.kimai.cloud

**Comment by Michael-Schaer** [comment]
Ahh my bad. I googled for "kimai2", which showed me the github page first. For some reason i thought i already checked if there was a website and there was none. I propably just confused it with another github repo. 🙄

But the cloud looks good and it's good to know you are hosting it on hetzner / germany 👍 

I'll check out the demo.
